### ISSUE TITLE
Need a way of dealing with forked projects. 

### ISSUE THREAD

**Issue by willwade** [openassistive/OpenATFrontEnd#34]
It’s bothering me that we are going to get forked/remixes of projects / improvements on projects. For example see the volkswitch. We really should put in the original project that it was based on. This could get messy with lots of similar titles and names. Need to think how best to deal with this in the UI.

**Comment by hosseinzoda** [comment]
At the time i believe dealing with it manually is good enough. Like adding in suffix,  "BarFork" to short title.
And it's possible to link them to each other with markdown.

**Comment by willwade** [comment]
Yeah agree - although going to leave this open to continue the discussion.  Your right - a simple link in the body to the original would suffice for now. 

I’ll just leave this here for now as it’s “related” ;) https://gohugo.io/content-management/related/
